packages/docusaurus/src/client/App.tsx

```diff
@@ -17,6 +17,35 @@ import PendingNavigation from './PendingNavigation';
 
 import './client-lifecycles-dispatcher';
 
+function isAnyCSSLoaded() {
+  const stylesheets = document.styleSheets;
+  for (let i = 0; i < stylesheets.length; i += 1) {
+    try {
+      if (
+        stylesheets[i].cssRules != null &&
+        stylesheets[i].cssRules.length >= 0
+      ) {
+        return true;
+      }
+    } catch (e) {}
+  }
+  return false;
+}
+
+function CSSLoadingWarningMessage({isClient}) {
+  if (isClient && (typeof window === 'undefined' || !isAnyCSSLoaded())) {
+    return (
+      <div style={{border: 'solid red thick', padding: '16px'}}>
+        <span>
+          You site CSS did not load properly. Your baseUrl setting is probably
+          bad.
```

Comment:
Still think we need a suggestion here. if we can tell the user the exact value he should use it's better.

Reply:
Agreed!
Maybe I can do something like `location.pathname.split('/')[0]` ?

You had also mentioned "only display this message on the homepage".
So should I maybe move this to somewhere else?